partials/footer.php

```diff
@@ -20,7 +21,11 @@
 		</a>
 		<p>©
 			<?php echo date("Y"); ?>
-			<?php echo get_bloginfo('name'); ?>. All rights reserved.
+		
+			<?php echo get_bloginfo('name'); ?>. 
+			<br/>
+			
+			All rights reserved.
```

Comment:
_All rights reserved._ text can go, this isn't needed

Reply:
Removed it.